Week-4/exercise_4.1/index.js

```diff
@@ -0,0 +1,30 @@
+const { log, error } = require('console')
+
+const getNumber = new Promise((resolve, reject) => {
```

Comment:
One thing here we need to create our own custom promise and use it.. not from inbuilt javascript promise